pkg/deploy/generator/util.go

```diff
@@ -3,9 +3,48 @@ package generator
 // Copyright (c) Microsoft Corporation.
 // Licensed under the Apache License 2.0.
 
+import (
```

Comment:
I am -1000 on util.go files.  Each of the functions below is called exclusively from one .go file.  Move each function in question into that .go file please.  Including max.